testdata/templates/common.tpl

```diff
@@ -7,9 +7,9 @@
 {{- define "email1" }}kyle@opslevel.com{{ end }}
 {{- define "email2" }}edgar@opslevel.com{{ end }}
 {{- define "email3" }}matthew@opslevel.com{{ end }}
-{{- define "id1" }}MTIzNDU2Nzg5MTIzNDU2Nzg5{{ end }}
-{{- define "id2" }}OTg3NjU0MzIxOTg3NjU0MzIx{{ end }}
-{{- define "id3" }}OTg3NjU0Mzg5MTIzNjU0MzIx{{ end }}
+{{- define "id1" }}Z2lkOi8vMTIzNDU2Nzg5OTg3NjU0MzIx{{ end }}
```

Comment:
these needed to be changed so they were proper IDs that properly pass through the `IsID` function and get properly detected as IDs vs Alias